src/components/pagination/Pagination.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import styles from './pagination.module.scss'; 
+import Right from '../../assets/images/icon-caret-right.svg?react';
+import Left from '../../assets/images/icon-caret-left.svg?react';
+
+// TODO: functionalitate butoane
+export default function Pagination () {
+    const [viewportWidth, setViewportWidth] = useState(window.innerWidth); 
```

Comment:
You don't need this. I see you are using it to detect mobile and show the labels `Prev` and `Next`. This can be handled through css if you wrap the text into a `span` that has a class that you control the display with media queries.

Now, since we said mobile first, the mentioned class should have the display: none by default and then, in the already existing media query you switch the display to `inline`/`inline-block`.

As a rule of thumb, whatever computation is needed for showing / hiding elements works a lot faster with css rather than javascript. JS should be the last resort in this kind of cases, where we hide elements based on props or other js variables.

Reply:
You're right 😁
truth be told, I don't know why I did it this way 😅.
I will change it.